core/utils.py

```diff
@@ -77,6 +85,7 @@ def random_poi(
     label_column: str,
     label_value: Any = -1,
     drop_label: bool = True,
+    seed: Optional[int] = None,
```

Comment:
Nit: Consider changing this to `random_seed` for consistency across the codebase.